tools/lumpy/lumpy.xml

```diff
@@ -9,64 +9,53 @@
         <exit_code range="1:" level="fatal" description="Tool exception" />
     </stdio>
     <command detect_errors="exit_code"><![CDATA[
+        #import re
+        #set one_sample_bam = re.sub('[^\w\-]', '_', str($analysis_type.input_file.element_identifier))
         #if $analysis_type.analysis_type_list == "one_sample":
-            ln -f -s '$analysis_type.input_file' input.bam &&
+            ln -f -s '$analysis_type.input_file' '$one_sample_bam' &&
         #else:
-            ln -f -s '$analysis_type.input_file' input.A.bam &&
-            ln -f -s '$analysis_type.input_fileB' input.B.bam &&
+            #set sample_a_bam = re.sub('[^\w\-]', '_', str($analysis_type.input_file.element_identifier))
+            #set sample_b_bam = re.sub('[^\w\-]', '_', str($analysis_type.input_fileB.element_identifier))
+            #if $sample_a_bam == $sample_b_bam:
+                #set sample_a_bam = "%s_a" % str($sample_a_bam)
+                #set sample_b_bam = "%s_b" % str($sample_b_bam)
+            #end if
```

Comment:
> Can you explain the purpose on this code expansion please ?

The above part makes sure we get correct names in the evidence column. That should help when you combine the VCFs from multiple comparisons.
This part here just makes sure that the sample identifiers are different (which is not essential, I can remove this if you prefer).

> I am not saying that we should not cope with that, but if on the other hand, the purpose of this is a twist to get the test for 2 samples running with a single test file, this is weird !

Just think of our bowtie wrapper, which always produces the output "Bowtie Output". I would consider this defensive programming. FWIW, you can't produce collections that have the same element identifier. Yet, for someone that just quickly throws in 2 datasets to check that the wrapper is working I think this is essential. If this was just to get the test data going I would be better of symlinking the same file as file_a.bam and file_b.bam. Your call.